power.mss

```diff
@@ -53,3 +53,30 @@
     marker-width: 3;
   }
 }
+
+#power-portals-way {
+  [zoom >= 16] {
+    line-width: 1.5;
+    line-color:#5B5B5B;
+  }
+}
+#power-portals-node {
+  [zoom >= 16] {
+    marker-file: url('symbols/man_made/power_tower_small.svg');
+    marker-width: 5;
+  }
+}
+#power-insulators-node {
```

Comment:
On the other hand, this is a detail that does not hurt general rendering, since it's related only to the power structure. Maybe we can test also some different rendering of them, but for me dots are OK (maybe some different color would work, I don't know).

Reply:
It _does_ hurt if the resulting rendering is missleading (see my second point) or if the resulting rendering is inconsistent (see my first point – that’s not the fault of @gustavecha but of the tagging scheme).

Reply:
I see. What do you think about "empty" dots then, for example?

Reply:
Insulators has been removed, so it's ok to resolve this point